test/index.js

```diff
@@ -49,17 +52,22 @@ describe('fs', () => {
     });
   });
 
-  it('exists() - path is required', () => {
-    fs.exists.should.to.throw('path is required!');
+  it('exists() - path is required', async () => {
+    try {
+      await fs.exists();
```

Comment:
The new code will terminate normally if no exception occurs.
In chai.js (without chai-as-promised), it is hard to write tests for exceptions in asynchronous processing.
The easiest way would be to call `should.fail()` immediately after the line where the exception should be raised.